<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QWidget" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,2,0,0,0,0,0,0,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="StrongBodyLabel" name="workerCharacterL">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>角色操作框</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ComboBox" name="workerCharacter"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="PushButton" name="characterNew">
         <property name="text">
          <string>新建角色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PrimaryPushButton" name="characterRemove">
         <property name="text">
          <string>删除角色</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignCenter</set>
       </property>
       <item row="1" column="0">
        <widget class="BodyLabel" name="workerIDL">
         <property name="text">
          <string>员工编号</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="LineEdit" name="workerID"/>
       </item>
       <item row="2" column="0">
        <widget class="BodyLabel" name="workerNameL">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>姓名</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="LineEdit" name="workerName"/>
       </item>
       <item row="3" column="0">
        <widget class="BodyLabel" name="workerAgeL">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>年龄</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="LineEdit" name="workerAge"/>
       </item>
       <item row="4" column="0">
        <widget class="BodyLabel" name="workerGenderL">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>性别</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="LineEdit" name="workerGender"/>
       </item>
       <item row="5" column="0">
        <widget class="BodyLabel" name="workerPlaceL">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>工作地点</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="LineEdit" name="workerPlace"/>
       </item>
       <item row="6" column="0">
        <widget class="BodyLabel" name="workerGroupL">
         <property name="text">
          <string>所属组织</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="ComboBox" name="workerGroup"/>
       </item>
       <item row="7" column="0">
        <widget class="BodyLabel" name="workerUserNameL">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用户名</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="LineEdit" name="workerUserName"/>
       </item>
       <item row="9" column="0">
        <widget class="BodyLabel" name="workerPasswordL">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="PasswordLineEdit" name="workerPassword"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="TitleLabel" name="label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>员工角色</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PushButton" name="characterAdd">
         <property name="text">
          <string>为员工添加角色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PrimaryPushButton" name="characterDelete">
         <property name="text">
          <string>为员工删除角色</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="ListWidget" name="characterList"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="PushButton" name="workerNew">
         <property name="text">
          <string>新建员工</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PushButton" name="workerUpdate">
         <property name="text">
          <string>修改员工信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PrimaryPushButton" name="workerRemove">
         <property name="text">
          <string>删除员工</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="ToolButton" name="refresh">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="PushButton" name="groupNew">
         <property name="text">
          <string>新建组织</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PrimaryPushButton" name="groupRemove">
         <property name="text">
          <string>删除组织</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="VerticalSeparator" name="VerticalSeparator"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="TreeWidget" name="groupTree">
       <attribute name="headerVisible">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>类型</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PasswordLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>VerticalSeparator</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
